Resources/views/Collector/db.html.twig

```diff
@@ -144,6 +144,12 @@
         .time-container { position: relative; }
         .time-container .nowrap { position: relative; z-index: 1; text-shadow: 0 0 2px #fff; }
         .time-bar { display: block; position: absolute; top: 0; left: 0; bottom: 0; background: #e0e0e0; }
+        .sql-formatted, .sql-runnable { background: var(--base-1, #f5f5f5); margin: .5em 0; padding: 1em; }
```

Comment:
`var(--base-1)` for 4.3+, fallback to `#f5f5f5` for 3.4 (that's the value there).